lib/y2_basetest.pm

```diff
@@ -10,15 +10,16 @@
 # Summary: This module provides common subroutines for YaST2 modules in graphical and text mode
 # Maintainer: Joaquín Rivera <jeriveramoya@suse.com>
 
-package y2_common;
+package y2_basetest;
```

Comment:
As we have discussed, `y2_basetest` is too generic. We should reflect that this is parent class for YaST tests in the installed system. For example `y2_module_basetest`. Just to confirm that it's not parent of `y2_installbase`.

Reply:
fixed